src/cluster/placement/service/mirrored_custom_groups_test.go

```diff
@@ -279,6 +283,15 @@ func newInstanceWithID(id string) placement.Instance {
 	)
 }
 
+var curInstanceNo = 0
+
+// Not thread safe; factor into a factory object if you need that.
```

Comment:
could you just make it thread safe - treat the var as atomic int64. 

will be a lot easier to not wonder if this is going to mess up based on incorrect code usage / multiple parallel tests /etc

Reply:
:D lol yeah good point, may as well.